packages/web-components/fast-foundation/src/carousel/carousel.template.ts

```diff
@@ -0,0 +1,104 @@
+
+const playIcon: string = `<svg viewBox="0 0 16 16" width="16px" height="16px" xmlns="http://www.w3.org/2000/svg"><path d="M1 0V16L15.0083 8L1 0Z"></path></svg>`;
+
+const tabbedTemplate: ViewTemplate = html<Carousel>`
+    <fast-tabs
+        class="carousel-tabs"
+        activeindicator="false"
+        activeid="${(x, c) => x.activeId}"
+        notabfocus="${x => (!x.paused ? "true" : "false")}"
+    >
+        ${repeat(
+            x => x.filteredItems,
+            html<Carousel>`<span
+                slot="tab"
+                id="tab-${(x, c) => c.index + 1}"
+                class="slide-tab"
+            ></span>`,
+            { positioning: true }
+        )}
+        ${x => html<Carousel>`
+            ${x.filteredItems.map(
+                (item: HTMLElement, index) =>
+                    `<div slot="tabpanel" aria-hidden="true" id="${tabPanelPrefix}${
+                        index + 1
+                    }" class="slide-container" role="${
+                        x.tabbed ? "tab-panel" : "group"
+                    }" ${x.tabbed ? "" : "aria-roledescription='slide'"}
+                    >
+                        ${item.outerHTML}
+                    </div>`
+            )}
+        `}
+        ${/*
+            // TODO: ASK This did not work as it shows 'x => x.outHTML' as a string as the child of the <div> when rendered. Should it work?
+
+        x => html<Carousel>`
+            ${repeat(
+                x => x.filteredItems,
+                html<Carousel>`<div slot="tabpanel" aria-hidden="true" id="${tabPanelPrefix}${
+                    (x,c) => c.index + 1
+                }" class="slide-container" role="${
+                    x => x.tabbed ? "tab-panel" : "group"
+                }" ${
+                    x => x.tabbed ? "" : "aria-roledescription='slide'"
+                    }
+                >
+                    ${x => x.outerHTML}
+                </div>`,
+                { positioning: true }
+            )}
+                `*/ ""}
```

Comment:
Follow up, we tried to get this working, but will need to do a deep dive as the repeat directive was not seeing the correct number of forwarded slots. For now Carousel has been rebuilt using a custom template, instead of implementing *Tabs* in the template.